### ISSUE TITLE
Weird crash involving a sheep and a lamprey from Better Animals Plus

### ISSUE THREAD

**Issue by MrRedstoneA1** [Liahim85/MistyWorld_Open#116]
I was walking around in the Overworld when the game suddenly froze and didn't respond. I looked at the log and the game had crashed and created a crash report, which I will paste below. All I know about the crash, according to the crash report, is that a sheep had a lamprey attached to it and attempted to eat some grass. The crash report suspects that the culprit was Misty World, but I also made a report with Better Animals Plus.
Minecraft 1.12.2
Forge 14.23.5.2860
Misty World 1.2.5.3
Better Animals Plus 9.0.1
[crash-2022-10-15_21.33.36-server.txt](https://github.com/Liahim85/MistyWorld_Open/files/9792887/crash-2022-10-15_21.33.36-server.txt)

**Comment by Liahim85** [comment]
Yes, obviously it's a misty world crash.  But now I have no way to fix it.  Perhaps I will do it later.
